File: Modul 6/C/PRAK605-2310817310010-AkmallullailSyaban.c
#include <stdio.h>

int main (){
    int n;
    scanf("%d", &n);
    int matriksA[n][n], result2[n][n];
    printf("Matriks A\n");
    for (int i = 0; i < n; i++){
        for (int j = 0; j < n; j++){
        scanf("%d", &matriksA[i][j]);
        }
    }
    printf("Matriks B\n"); 
    int matriksB[n][n];
    for (int i = 0; i < n; i++){
        for (int j = 0; j < n; j++){
        scanf("%d", &matriksB[i][j]);
        }
    }
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            int result = 0;
            for (int k = 0; k < n; k++) {
                result += matriksA[i][k] * matriksB[k][j];
            }
            result2[i][j] = result;
        }
    }
    printf("Matriks AXB\n");
    for (int i = 0; i < n; i++){
        for (int j = 0; j < n; j++){
            printf("%d ", result2[i][j]);
    }
    printf("\n");
    }
    return 0;
}

File: Modul 6/Python/PRAK605-2310817310010-AkmallullailSyaban.py
n = int(input())
matriksA = []
result2 = []

print("Matriks A")
for i in range(n):
    row = list(map(int, input().split()))
    matriksA.append(row)

print("Matriks B")
matriksB = []
for i in range(n):
    row = list(map(int, input().split()))
    matriksB.append(row)

for i in range(n):
    result2.append([0] * n)

for i in range(n):
    for j in range(n):
        result = 0
        for k in range(n):
            result += matriksA[i][k] * matriksB[k][j]
        result2[i][j] = result

print("Matriks AXB")
for i in range(n):
    for j in range(n):
        print(result2[i][j], end=" ")
    print()